website/docs/docs/on-premises/releases/1.26.0.md

```diff
@@ -0,0 +1,27 @@
+---
+title: 1.26.0
+---
+
+## Release date
+
+07 August 2023
+
+## Features
+
+* Support application/problem+json error response format for all non-200 responses
+* null
```

Comment:
bah! i need to fix those in the script, but I think for the moment, we just clean this up

Reply:
I'm all for releasing null.